packages/cli/src/cmds/validator/options.ts

```diff
@@ -152,6 +161,12 @@ export const validatorOptions: ICliCommandOptions<IValidatorCliArgs> = {
     type: "string",
   },
 
+  flushKeymanagerProposerConfigs: {
+    description: "If keymanager's persisted proposer configs are found, delete them all!",
```

Comment:
This is a very strange flag, is this really necessary?

Reply:
this to to clear keymanager proposer configs, although name can be better :slightly_smiling_face: , any suggestions?

Reply:
What's the usecase? There's the delete API for that right?

Reply:
the use case is you used keymanager api to make proposer data updates, and now was to discard it and switch to proposerFileConfig. Since now `proposerFileConfig` flag will error since we don't accept `proposerFileConfig` is there is keymanager persisted proposer data, this flag can be used to discard and clean keymanagaer persisted configs.

Reply:
removed the flag! and recommended manual deletion in case of conflict :+1: